README.md

```diff
@@ -168,7 +168,21 @@ post.redo!
 post.switch_to!(2)
 ```
 
-If you update record after `#undo!` or `#switch_to!` you lose all "future" versions and `#redo!` is no longer possible.
+Normally, if you update record after `#undo!` or `#switch_to!` you lose all "future" versions and `#redo!` is no
+longer possible. However, you can provide an `as_new: true` option to `#undo!` or `#switch_to!`, which will
+create a new version with old data. Caveat: when switching to a newer version, `as_new` will have no effect.
+
+```ruby
+post = Post.create!(title: 'first post') # v1
+post.update!(title: 'new title')         # v2
+post.undo!(as_new: true)                 # v3 (with same attributes as v1)
```

Comment:
> Besides, you're the one who said:

> > We don't want to build Git-over-PostgreSQL.

Good catch)

`copy` sounds ambiguous to me, sounds like we want to copy all attributes os smth like that. Maybe `append: true`? "Append changes to the log". Sounds more log-centric.

Btw, we should add this option to `switch_to!` method as well.
And also add a note that `redo!` will not work if you use `undo!(append: true)`, 'cause there are no future versions.